src/gei/Services/GhesVersionCheckerService.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Threading.Tasks;
+using OctoshiftCLI.Extensions;
+
+namespace OctoshiftCLI.GithubEnterpriseImporter.Services
+{
```

Comment:
nit: convert to file scoped namespace

Reply:
done https://github.com/github/gh-gei/pull/924/commits/e39386d888d7451ff3c747fe3a1d0badd7a73c4d